source/GameAction.cpp

```diff
@@ -214,6 +215,16 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "relocate")
+	{
+		for(const DataNode &grand : child)
+		{
+			if(grand.Token(0) == "planet" && grand.Size() > 1)
+				teleportPlanet = GameData::Planets().Get(grand.Token(1));
+			if(grand.Token(0) == "flagship only")
```

Comment:
These are meant to be on separate lines? I think they should be in the same argument or as a sub section of it at worst.
Did someone else suggest this?

Reply:
suggestion by warp-core and pertervdmeer

Reply:
https://github.com/endless-sky/endless-sky/pull/7348#discussion_r1005007433

Reply:
Fair then!